File: c/linearsearch.c
#include <stdio.h>
void linear(int a[], int n, int key)
{
    int i, f = 0;
    for (i = 0; i < n; i++)
    {
        if (a[i] == key)
        {
            f = 1;
            break;
        }
    }
    if (f == 1)
    {
        printf("key element is found at position %d\n", i + 1);
    }
    else
    {
        printf("key element is not found\n");
    }
}
void main()
{
    int a[100], i, n, key;
    printf("how many elements in your array : ");
    scanf("%d", &n);
    for (i = 0; i < n; i++)
    {
        printf("enter element  : ");
        scanf("%d", &a[i]);
    }
    printf("\nenter key element : ");
    scanf("%d", &key);
    linear(a, n, key);
}

File: FC_Java_InterviewQues/linearSearch.java
package FC_Java_InterviewQues;

public class linearSearch {
	public int Linear(int a[],int key) {
		for(int i=0;i<a.length;i++)
		{	
			if(a[i]==key)
			return i;
		} 
			return 0;
		
	}
	public static void main(String[] args) {
		int a[]= {1,2,3,4,5};
		int key = 5;
		
		linearSearch ls=new linearSearch();
		int n=ls.Linear(a, key);
		if(n==1) {
			System.out.println("Element is present at "+n+"position");
		}
		else {
			System.out.println("Not Present");
		}
		
	}

}
